x/schema/keeper/schema.go

```diff
@@ -1,14 +1,62 @@
 package keeper
 
 import (
+	"bytes"
 	"encoding/binary"
+	"encoding/json"
+	"os"
+	"path/filepath"
 
 	"github.com/cosmos/cosmos-sdk/store/prefix"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/google/uuid"
+	shell "github.com/ipfs/go-ipfs-api"
 	"github.com/sonr-io/sonr/x/schema/types"
 )
 
+const (
+	URL_PERSISTENCE_READ = "ipfs.Sonr.ws"
+)
```

Comment:
if not a ton of effort, pull from env?

Reply:
You can add it and read from viper config at cmd/sonrd/main.go